src/services/login/create-account-service.ts

```diff
@@ -0,0 +1,28 @@
+import * as bodyparser from 'body-parser';
+import * as express from 'express';
+import { NextFunction, Request, Response, Router } from 'express';
+import * as httpserver from 'http';
+const Web3 = require('web3');
+const web3 = new Web3(new Web3.providers.HttpProvider('HTTP://127.0.0.1:7545'));
+
+export async function createNewAccount(privateKey: String) {
+	// this section generates the account
+	const newAccountAddress = await web3.eth.personal.importRawKey(
+		privateKey,
+		'password'
+	);
+	const accounts = await web3.eth.getAccounts();
+	const newAccountIndex = accounts.length - 1;
+	const newAccount = accounts[newAccountIndex];
+

```

Comment:
Wasn't there some code to unlock the accounts based on some words, is that not needed anymore.

Reply:
The code to unlock accounts has to be placed where ever the contracts are deployed, so when updating the info on blockchain, i can put it in if you direct me to the scripts where contracts are called. Also the code that needs to be inserted is in the comments below in this file

Reply:
Okay Ill look at it after merge.